Author: Antonio Terceiro <terceiro@debian.org>
Description: set kmsg=0 for centos as well
 Without this, system-journald will stay at 100% of CPU all the time.

--- a/templates/lxc-centos.in
+++ b/templates/lxc-centos.in
@@ -604,6 +604,8 @@ lxc.utsname = $utsname
 
 lxc.autodev = $auto_dev
 
+lxc.kmsg = 0
+
 # When using LXC with apparmor, uncomment the next line to run unconfined:
 #lxc.aa_profile = unconfined
 
--- a/config/templates/centos.common.conf.in
+++ b/config/templates/centos.common.conf.in
@@ -28,8 +28,9 @@ lxc.hook.clone = @LXCHOOKDIR@/clonehostn
 # lxc.cap.drop = audit_control    # breaks sshd (set_loginuid failed)
 # lxc.cap.drop = audit_write
 # lxc.cap.drop = setpcap          # big big login delays in CentOS 7 systemd
+# lxc.cap.drop = setfcap          # breaks installing packages with setuid binaries
 #
-lxc.cap.drop = mac_admin mac_override setfcap
+lxc.cap.drop = mac_admin mac_override
 lxc.cap.drop = sys_module sys_nice sys_pacct
 lxc.cap.drop = sys_rawio sys_time
 
